browser/src/canvas/sections/OtherViewCellCursorSection.ts

```diff
@@ -180,11 +184,22 @@ class OtherViewCellCursorSection extends CanvasSectionObject {
     public static updateVisibilities() {
         for (let i = 0; i < OtherViewCellCursorSection.sectionPointers.length; i++) {
             const section = OtherViewCellCursorSection.sectionPointers[i];
-            section.setShowSection(section.checkMyVisibility());
+            const newState = section.checkMyVisibility();
+
+            if (newState !== section.showSection) {
```

Comment:
don't we have dedicated function for that? `onSectionShowStatusChange`

https://github.com/CollaboraOnline/online/blob/3b07ba7262bd0749a4bd35dca1d8cde6deeb5000/browser/src/canvas/CanvasSectionContainer.ts#L1967

Possibly that `setShowSection` could also guard against calling `onDocumentObjectVisibilityChange` without good reason

Reply:
That's right we can add a guard and check its usages. But here we are also checking if we need to hide the popups.

Reply:
I mean we can put exactly that there:
```
onSectionShowStatusChange() {
          if (newState === false)
                    section.hideUsernamePopUp();
}
```

Reply:
Aha, yes we could :) Let me keep that in mind. I didn't test it. But we can also use that callback i guess.